### PR TITLE
hub: fix arch detection of mock configs for `wrapperRPM` tasks

### PR DESCRIPTION
**Pull request by lzaoral**
Fixes: 8bb2b03 ("hub: fix generation of mock configs for noarch builds)
Related: https://issues.redhat.com/browse/OSH-69
Related: https://issues.redhat.com/browse/OSH-538

### CODE REVIEW

File: osh/hub/scan/mock.py [hunk 1]
```diff
@@ -116,23 +116,30 @@ def _get_build_arches(koji_proxy, task_id, tag):
...
         child = koji_proxy.getTaskInfo(child_id, request=True)
 
         # skip non-build subtasks
         method = child['method']
-        if method != 'buildArch':
+        if method not in ('buildArch', 'wrapperRPM'):
             continue
 
+        # wrapperRPM tasks do not track arches directly
+        if method == 'wrapperRPM':
```
Comment (kdudka) [inline osh/hub/scan/mock.py:134]: If this was inserted before the previous condition, the other condition could stay as it is.

**Review by kdudka — APPROVED** [review]
Looks good.  Thanks for the quick fix!

File: osh/hub/scan/mock.py (hunk 1 above)
Comment (lzaoral) [reply osh/hub/scan/mock.py:134]: Good idea!

Comment (lzaoral) [reply osh/hub/scan/mock.py:130]: Addressed in 2fe56b90291e32119ef69f8689b428c7424d1f44.

**Pull request merged**